docs/src/main/asciidoc/deploying-to-openshift-docker-howto.adoc

```diff
@@ -0,0 +1,154 @@
++
+[source,shell,subs="+quotes",options="nowrap"]
+----
+oc get pods
+----
++
+[source,shell,subs="+quotes",options="nowrap"]
+----
+NAME                            READY   STATUS      RESTARTS   AGE
+openshift-helloworld-1-build    0/1     Completed   0          11m
+openshift-helloworld-1-deploy   0/1     Completed   0          10m
+openshift-helloworld-1-gzzrx    1/1     Running     0          10m
+----
+
+. To retrieve the log output for your application's pod, use the `oc logs -f` command with the `<pod_name>` value of the pod you are interested in.
+In this example, we use the `openshift-helloworld-1-gzzrx` pod name that corresponds with the latest pod prefixed with the name of your application:
++
+[source,shell,subs="+quotes",options="nowrap"]
+----
+oc logs -f _openshift-helloworld-1-gzzrx_
+----
++
+[source,shell,subs=attributes+]
+----
+Starting the Java application using /opt/jboss/container/java/run/run-java.sh ...
+INFO exec -a "java" java -Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager -XX:MaxRAMPercentage=50.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -XX:+ExitOnOutOfMemoryError -cp "." -jar /deployments/quarkus-run.jar
+__  ____  __  _____   ___  __ ____  ______
+--/ __ \/ / / / _ | / _ \/ //_/ / / / __/
+-/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \
+--\___\_\____/_/ |_/_/|_/_/|_|\____/___/
+2024-09-17 10:23:25,254 INFO  [io.quarkus] (main) getting-started 1.0.0-SNAPSHOT on JVM (powered by Quarkus {QuarkusCore}) started in 0.653s. Listening on: http://0.0.0.0:8080
+2024-09-17 10:23:25,281 INFO  [io.quarkus] (main) Profile prod activated.
+2024-09-17 10:23:25,281 INFO  [io.quarkus] (main) Installed features: [cdi, kubernetes, rest, smallrye-context-propagation, vertx]
+----
+
+. Retrieve a list of services:
++
+[source,shell,subs="+quotes",options="nowrap"]
+----
+oc get svc
+----
++
+[source,shell,subs="+quotes",options="nowrap"]
+----
+NAME                   TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                               AGE
+openshift-helloworld   ClusterIP   172.30.64.57     <none>        80/TCP                                14m
+----
+
+. Get a URL to test your application.
++
+[NOTE]
+====
+To create an {openshift} route, ensure you have specified `quarkus.openshift.route.expose=true` in the `application.properties` file.
+====
++
+[source,shell,subs="+quotes",options="nowrap"]
+----
+oc get routes
+----
++
+[source,shell,subs="+quotes",options="nowrap"]
+----
+NAME                   HOST/PORT                                                                   PATH   SERVICES               PORT   TERMINATION   WILDCARD
+openshift-helloworld   openshift-helloworld-username-dev.apps.sandbox-m2.ll9k.p1.openshiftapps.com          openshift-helloworld   http                 None
+----
++
+[NOTE]
+====
+Be aware that the route is now listening on port 80 and no longer at port 8080.
```

Comment:
I'd double-check with the SME correctness of  the current statement. Is the route is listening on port 80 _in addition to_ port 8080? If so, maybe:
```suggestion
Be aware that the route is now listening on port 80.
```

Reply:
@iocanel, perhaps you could advise here?